GCMessengerSDKSample/GCMessengerSDKSample/AccountDetailsViewController.swift

```diff
@@ -48,6 +48,7 @@ class AccountDetailsViewController: UIViewController {
             present(alert, animated: true)
 
         } else {
+            loggingSwitch.setOn(false, animated: false) // TODO: GMDX-1514 this is a check to see if Transport stop crashing app
```

Comment:
@swienribGenesys, since this comment is publicly visible, it would be better to express it more clearly and professionally. Here's a suggestion:

"This code serves the purpose of detecting any issues related to Transport that may lead to app crashes. Note that this task is currently pending under GMDX-1514."